### ISSUE TITLE
video player not shown in some cases on Android

### ISSUE THREAD

**Issue by maoamid** [eclipsesource/tabris-js#1424]
### Problem description
On Android:  Video player is not initialized in some layouts like bellow
On Android and iOS the player always constrain the video w/h proportions. This is a forced behavior?
Also the player load time is about 2-3 seconds, before the video file start to cache.
On iOS - the console throw an error fixed by:
```javascript
 }).on('stateChanged', event => button.text = event.value !== 'pause' ? '❚❚' : '▶')
```
should become: 
```javascript
}).on('stateChanged', event => this.find('.button').first.text = event.value !== 'pause' ? '❚❚' : '▶')
```

### Expected behavior
Video player should load quickly and be displayed at least

### Environment

- Tabris.js version: "nightly" tabris@2.0.0-rc2-dev.20170711
- Device: Lenovo && iPhone 5S
- OS: Android 4.4 && iOS 

### Code snippet

```javascript
module.exports = class test extends Composite {
constructor(properties) {
        super(Object.assign({ id: 'body', top: 0, bottom: 0, left: 0, right: 0 }, properties));
                this._displayVideoPlayer();
    }

    _displayVideoPlayer() {

        let video = new Video({
        left: 0, top: 0, right: 0,
        background: 'red',
        url: 'http://peach.themazzone.com/durian/movies/sintel-1280-stereo.mp4',
        controlsVisible: false
        }).on('stateChanged', event => button.text = event.value !== 'pause' ? '❚❚' : '▶')
        .appendTo(this);

        let button = new Button({
        id: 'button',
        centerX: 0, bottom: 16,
        text: '❚❚'
        }).on('select', () => video.state === 'play' ? video.pause() : video.play())
        .appendTo(this);
    }

}
```

**Comment by ralfstx** [comment]
The error message on iOS is fixed with f314e46.

**Comment by maoamid** [comment]
@ralfstx I don't agree with your solution
 1. If I modify the snippet so the button 'overlap' the video (to create some controls button for example, on top of the video), you see that the button is not visible, instead, the tap event is received in that area, even the  Z index of video is higher. 
2. Also using the logic, this snippet is very confusing, because the video widget layout should never be relative to his controls ui elements, but because we cannot append widgets to video, it's normal to attach the video and his custom controls both to a parent creating a 'video component'. I understand also that this snippet is just a basic one, just for showing some basics, so maybe my this second point are not really an issue, sorry ;)

**Comment by ralfstx** [comment]
@maoamid I believe the solution in f314e46 is correct. The button does not overlap the video – we try to keep the snippets simple.

If you want buttons on top of the video, that would be a different snippet. If you have problems creating this snippet, let's discuss this on Slack or StackOverflow.

So this issue is basically about load time on Android, right?

**Comment by maoamid** [comment]
@ralfstx  yes it's about load time, and also about not displaying the video at all,  using the snippet from my first comment here (only on Android)
thank you for looking on this.
